### ISSUE TITLE
Ingen executes action on next message after getting message at a high rate

### ISSUE THREAD

**Issue by loki42** [drobilla/ingen#37]
I've got a script to reproduce this but it requires a specific Ingen patch etc. Basically, I play a sound through Ingen and record the level of the output after I mute and unmute it. 

The flow of the script it:
I set a bunch of parameters, set them back to what they were. Sleeping between each parameter "delay" amount.
mute by setting VCA level, check output level by recording through jack.
unmute by setting VCA level and check level
if mute level is to loud, or unmute level is to quiet, print error.

If I run my script, I get no errors with longer delay times but when I start reducing it, I start to get errors. The behaviour when the error has been triggered is that actions are executed when the next message is received. For example, a disconnect won't happen straight away but will happen when the next set value is sent.

**Comment by drobilla** [comment]
> I've got a script to reproduce this but it requires a specific Ingen patch etc

I'd rather that than nothing.

**Comment by drobilla** [comment]
Come to think of it, a lot of the events sent will do, and should be easy to produce.

The description sounds like it's only about setting parameters, but then

> For example, a disconnect won't happen straight away but will happen when the next set value is sent

So I'm suspicious.

**Comment by loki42** [comment]
Yes, it does it for an message, not just setting parameters. Disconnect / connect / basically any message.
